### DISCUSSION TITLE
NPC 等级系统 - 晦涩弗里曼的博客

### DISCUSSION THREAD

**Discussion by giscus** [obscurefreeman/blog#10] [Giscus]
# NPC 等级系统 - 晦涩弗里曼的博客

如果 NPC 有自我意识

https://obscurefreeman.netlify.app/addons/2024-11-12-npc%20%E7%AD%89%E7%BA%A7%E7%B3%BB%E7%BB%9F/

**Comment by obscurefreeman** [comment]
更多效果图（Steam 创意工坊页面）
![img](https://steamuserimages-a.akamaihd.net/ugc/35560324660413175/B8F03EC110CF2340C38F5ED4DD26F22CD5503292/?imw=637&imh=358&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true)
![img](https://steamuserimages-a.akamaihd.net/ugc/35559600581654712/2EDA11294662025D32999C12E13CB614B39F8AE4/?imw=637&imh=358&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true)
![img](https://steamuserimages-a.akamaihd.net/ugc/35560324660412113/CBDD0532C0C954A31F617ABAF0D7BB182227788F/?imw=637&imh=358&ima=fit&impolicy=Letterbox&imcolor=%23000000&letterbox=true)
